dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/PowerSystemStabilizers/Standard/PssIEEE2B.mo

```diff
@@ -0,0 +1,150 @@
+    Placement(visible = true, transformation(origin = {-90, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Feedback feedback annotation(
+    Placement(visible = true, transformation(origin = {70, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.FirstOrder firstOrder(T = t6)  annotation(
+    Placement(visible = true, transformation(origin = {-50, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.FirstOrder firstOrder1(T = t7, k = Ks2)  annotation(
+    Placement(visible = true, transformation(origin = {-50, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Gain gain(k = Ks3)  annotation(
+    Placement(visible = true, transformation(origin = {-30, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 90)));
+  Modelica.Blocks.Math.Gain gain1(k = Ks1)  annotation(
+    Placement(visible = true, transformation(origin = {110, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Nonlinear.Limiter limiter(limitsAtInit = true, uMax = VstMax, uMin = VstMin) annotation(
+    Placement(visible = true, transformation(origin = {210, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Nonlinear.Limiter limiter1(limitsAtInit = true, uMax = Vsi1Max, uMin = Vsi1Min) annotation(
+    Placement(visible = true, transformation(origin = {-170, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Nonlinear.Limiter limiter2(limitsAtInit = true, uMax = Vsi2Max, uMin = Vsi2Min) annotation(
+    Placement(visible = true, transformation(origin = {-170, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Dynawo.NonElectrical.Blocks.Continuous.RampTrackingFilter rampTrackingFilter(M = 5, t1 = t8, t2 = t9)  annotation(
+    Placement(visible = true, transformation(origin = {30, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.TransferFunction transferFunction1(a = {t11, 1}, b = {t10, 1})  annotation(
+    Placement(visible = true, transformation(origin = {170, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.TransferFunction transferFunction2(a = {t4, 1}, b = {t3, 1})  annotation(
+    Placement(visible = true, transformation(origin = {130, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.TransferFunction transferFunction3(a = {t2, 1}, b = {t1, 1})  annotation(
+    Placement(visible = true, transformation(origin = {150, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Gain gain2(k = SystemBase.SnRef / SNom) annotation(
+    Placement(visible = true, transformation(origin = {-210, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+  //Init parameters
+  parameter Types.ActivePowerPu PGen0Pu "Initial active power input in pu (base SnRef) - generator convention";
```

Comment:
I usually write **(generator convention)**

I think the minus sign could be misinterpreted.